.github/workflows/provisioning-cli-s3.yml

```diff
@@ -1,10 +1,24 @@
 name: 'Provisioning: CLI S3 Distribution'
 
 on:
+  # This workflow is also part of the CLI release pipeline.
   workflow_call:
+    inputs:
+      deploy:
+        type: boolean
+        required: true
+  # Run tests in a PR to the main branch.
+  pull_request:
+    paths:
+      - provisioning/cli-dist/**
+    branches:
+      - main
+  # Run tests and deploy on merge to the main branch.
   push:
     paths:
       - provisioning/cli-dist/**
+    branches:
+      - main
```

Comment:
Tento CLI dist:
- uz spustame aj vramci CLI release (`workflow_call`)
- ale spustalo sa to aj kde to nie je potrebne, napr. release templates
- preto som to upravil na push a PR do `main`.